server/test.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var request = require("request");
+
+var host = "localhost";
+if(process.env.DOCKER_HOST) {
+  // Get the IP out of the DOCKER HOST if this machine is using docker machine or boot2docker.
+  var rx = /(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)/;
+  host = rx.exec(process.env.DOCKER_HOST)[0];
+}
+var port = process.env.PORT || 8000;
+
+var uri = "http://" + host + ":" + port + "/tile";
```

Comment:
That makes sense. In defense of the Ick, I never intended this to stick around ;)